utils/release_tracking.py

```diff
@@ -0,0 +1,97 @@
+import argparse
+import re
+import os
+import subprocess
+import sys
+
+
+def run_cargo_build(path):
+    print(f"cargo build, path: {path}")
+    command = f'cargo build'
+    subprocess.check_output(command.split(), cwd=path)
+
+
+def clone_repo(clone_dir, repo_name, branch):
+    git_repo = f"https://github.com/parallaxsecond/{repo_name}.git"
+    future_repo_dir = os.path.join(clone_dir, repo_name)
+    if not os.path.isdir(future_repo_dir):
+        command = f"git clone {git_repo} -b {branch} {future_repo_dir}"
+        subprocess.check_output(command.split())
+        command = f"git submodule update --init"
+        subprocess.check_output(command.split(), cwd=future_repo_dir)
+
+
+def git_toml_deps(toml_path, updatable_deps, deps_repos):
+    lines = None
+    with open(toml_path, 'r') as f:
+        lines = f.readlines()
+
+    for i in range(len(lines)):
+        line = lines[i]
+        for dep in updatable_deps.keys():
```

Comment:
We could instead do something like that:

    for dep, dep_name in updatable_deps.items():
    ....

Reply:
I only needed the keys of the dictionary, that's why I didn't use `items()`